src/app/components/CarouselCoins.tsx

```diff
@@ -66,13 +84,41 @@ export default function CarouselCoins() {
 
               <div className="flex sm:flex-col items-center sm:items-start gap-4 sm:gap-0">
                 <div className="text-[16px] dark:text-white text-[#181825] font-medium leading-6 text-left">
-                  <span className="hidden sm:inline">{coin.name}</span>{" "}
-                  <span className="uppercase text-sm font-medium sm:text-base">
+                  <span
+                    className={`${
+                      selectedCoin === coin.id && "text-white"
```

Comment:
You can take the check `selectedCoin === coin.id` in a variable because it is getting used at multiple places.